src/containers/SearchPageContainer.tsx

```
@@ -0,0 +1,95 @@
+`
+
+const CheckBoxElement = styled.input`
+  display: none;
+`
+
+const items = [
+  { label: '新品', name: 'new' },
+  { label: '中古', name: 'used' },
+];
+
+const SearchPageContainer = ({ params }: SearchPageProps) => {
```

ESLint

[error] 'params' is defined but never used.

(@typescript-eslint/no-unused-vars)